### PR TITLE
Create team-working-agreement

### LINKED ISSUE #3: add team info, working agreement, etc etc to repo
add a basic structure and information as a start

### PR DESCRIPTION
**Pull request by susanev**
@anita-trimbur @kimberleyamackenzie i made a best guess at your working hours, please let me know what they actually are and ill update them

is there anything missing from this that we have talked about and that i should add now?

fixes https://github.com/pulumi/user-experience/issues/3

### CODE REVIEW

File: team-working-agreement.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Team Meetings
+
+These are meant to be informal. We come together and chat about anything we'd like, work realted or not.
+This meeting style is inspire by [Honeycomb's approach](https://www.honeycomb.io/blog/standup-meetings-are-dead/).
+
+### Iteration Planning
+
+- before the start of the meeting everyone should review the backlog and ready columns and move things
+up to the top that are high impact or that folks have interest in working on next iteration
```
Comment (susanev) [inline team-working-agreement.md]:
```suggestion
- Before the start of the meeting everyone should review the backlog and ready columns and move things
up to the top that are high impact or that folks have interest in working on the next iteration
```

File: team-working-agreement.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+#### General Structure
+
+- celebrate what was completed last milestone
+- move anything started but unfinished to the next milestone
+- quick review of quarter epics and progress
+- discuss anything not-started from the last iteration and decided whether to move it to the next milestone
+- remove milestone filter and together add what we want to accomplish this milestone
```
Comment (susanev) [inline team-working-agreement.md]:
```suggestion
- Celebrate what was completed the last milestone
- Move anything started but unfinished to the next milestone
- A quick review of quarter epics and progress
- Discuss anything not-started from the last iteration and decided whether to move it to the next milestone
- Remove milestone filter and together add what we want to accomplish this milestone
```

File: team-working-agreement.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## General Working Hours
+
+Try and avoid scheduling team meetings outside of the shared range.
+
+- susan: 5 am pt – 3 pm pt
+- Kimberley: 8:30 am pt – 4:30 pm pt
+- Anita: 10:00 am et – 5:00 pm et
+
+## Team meetings
```
Comment (susanev) [inline team-working-agreement.md]:
```suggestion
## Meetings
```

File: team-working-agreement.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+- Team meeting: every Tuesday/Thursday 8:30 am pt – 9:00 am pt / 11:30 am et – 12:00 pm et
+- Iteration planning: first Monday of the iteration 8:30 am pt – 9:00 am pt / 11:30 am et – 12:00 pm et
+- Retro: last Friday of the iteration 8:30 am pt – 9:30 am pt / 11:30 am et – 12:30 pm et
+
+## Team Meetings
```
Comment (susanev) [inline team-working-agreement.md]:
```suggestion
### Team Meetings
```

File: team-working-agreement.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+#### General Structure
+
+- Celebrate what was completed the last milestone
+- Move anything started but unfinished to the next milestone
+- A quick review of quarter epics and progress
+- Discuss anything not-started from the last iteration and decided whether to move it to the next milestone
+- Remove milestone filter and together add what we want to accomplish this milestone
+
+## Retro
```
Comment (susanev) [inline team-working-agreement.md]:
```suggestion
### Retro
```

**Review by kimberleyamackenzie — APPROVED** [review]
🎉

File: team-working-agreement.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+## Meetings
+
+- Team meeting: every Tuesday/Thursday 8:30 am pt – 9:00 am pt / 11:30 am et – 12:00 pm et
+- Iteration planning: first Monday of the iteration 8:30 am pt – 9:00 am pt / 11:30 am et – 12:00 pm et
+- Retro: last Friday of the iteration 8:30 am pt – 9:30 am pt / 11:30 am et – 12:30 pm et
+
+### Team Meetings
+
+These are meant to be informal. We come together and chat about anything we'd like, work realted or not.
+This meeting style is inspire by [Honeycomb's approach](https://www.honeycomb.io/blog/standup-meetings-are-dead/).
```
Comment (susanev) [inline team-working-agreement.md]:
```suggestion
This meeting style is inspired by [Honeycomb's approach](https://www.honeycomb.io/blog/standup-meetings-are-dead/).
```

File: team-working-agreement.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+- Before the start of the meeting everyone should review the backlog and ready columns and move things
+up to the top that are high impact or that folks have interest in working on the next iteration
+
+#### General Structure
+
+- Celebrate what was completed the last milestone
+- Move anything started but unfinished to the next milestone
+- A quick review of quarter epics and progress
+- Discuss anything not-started from the last iteration and decided whether to move it to the next milestone
```
Comment (susanev) [inline team-working-agreement.md]:
```suggestion
- Discuss anything not-started from the last milestone and decided whether to move it to the next milestone
```

**Review by anita-trimbur — COMMENTED** [review]
@susanev Re: working hours. I'm generally online around 9AM EST. Idk if it's a worthwhile distinction, but while I prefer not to take meetings after 5PM, I do check Slack and putter around until closer to 6. Otherwise everything looks great!

File: team-working-agreement.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
+# Team Working Agreement
+
+## General Working Hours
+
+Try and avoid scheduling team meetings outside of the shared range.
+
+- susan: 5 am pt – 3 pm pt
+- Kimberley: 8:30 am pt – 4:30 pm pt
+- Anita: 10:00 am et – 5:00 pm et
```
Comment (susanev) [inline team-working-agreement.md]:
```suggestion
- Anita: 9:00 am et – 5:00 pm et
```

**Pull request merged**